dicomutil/dicomutil.go

```diff
@@ -19,11 +19,15 @@ import (
 )
 
 var (
-	printMetadata = flag.Bool("print-metadata", true, "Print image metadata")
-	extractImages = flag.Bool("extract-images", false, "Extract images into separate files")
-	verbose       = flag.Bool("verbose", false, "Activate high verbosity log operation")
+	printMetadata       = flag.Bool("print-metadata", true, "Print image metadata")
+	extractImages       = flag.Bool("extract-images", false, "Extract images into separate files")
+	extractImagesStream = flag.Bool("extract-images-stream", false, "Extract images using frame streaming capability")
+	verbose             = flag.Bool("verbose", false, "Activate high verbosity log operation")
 )
 
+// FrameBufferSize represents the size of the *Frame buffered channel for streaming calls
+const FrameBufferSize = 100
```

Comment:
pretty cheap since we're storing `*Frame`, shouldn't have over a 100 frames build up in the buffer.